types/IMBuildings/PeopleCounter/uplink.spec.js

```diff
@@ -80,15 +80,68 @@ describe("IMBuilding People counter", () => {
         assert.typeOf(value.data, "object");
 
         assert.equal(value.topic, "lifecycle");
-        assert.equal(value.data.deviceStatus, 8);
+        assert.equal(value.data.deviceStatus, "BATTERY_NOT_FULL");
         assert.equal(value.data.payloadCounter, 97);
-        assert.equal(value.data.sensorStatus, 32);
+        assert.equal(value.data.sensorStatus, "RECEIVER_LOW_BATTERY");
         assert.equal(value.data.voltage, 2.48);
 
         utils.validateSchema(value.data, lifecycleSchema, { throwError: true });
       });
 
       consume(data);
     });
+
+    it("should decode the IMBuilding People NB-IOT payload", () => {
+      const data = {
+        data: {
+          reports: [
```

Comment:
this is iotcreators specific right? if yes, how do we support other structures?

Reply:
As this is the default way. I suppose we keep it that way. I cant really work with structures i don't know

Reply:
We are going to provide an out of the box support for iotcreators which will simplify the setup